packages/runtime/src/api/createAPIv2.ts

```diff
@@ -854,6 +885,25 @@ export function createAPI(
         })
       }
     },
+    triggerActions: () => {
+      const apiData = ctx.dataSignal.get().Apis?.[api.name]
+      if (
+        apiData === undefined ||
+        (apiData.data === null && apiData.error === null)
+      ) {
+        return
+      }
+      if (apiData.error) {
```

Comment:
Does this take the error formula into account?

Reply:
Currently this works by applying the actions to the currently set data. Before the data is set on the signal the `isError` formula is applied. So I believe it is safe to assume that `error` is already correct. It is the same logic used when determining if its a success or error when hydrating from ssr apis.